apollo-ios/Sources/Apollo/ApolloClientProtocol.swift

```diff
@@ -27,7 +27,7 @@ public protocol ApolloClientProtocol: AnyObject {
   ///   - resultHandler: [optional] A closure that is called when query results are available or when an error occurs.
```

Comment:
`ApolloClientProtocol` will be updated once I've finalized the API for `ApolloClient` (or maybe moved into `ApolloTestHelpers`?)